src/utils/deprecate.js

```diff
@@ -25,7 +25,7 @@ const deprecate = function(message, test) {
 
 deprecate._console = typeof console !== 'undefined' ? console : {};
 deprecate._warn = function() {
-  const warn = deprecate._console.warn || deprecate._console.log || function() {};
+  const warn = deprecate._console.warn || deprecate._console.log || Function.prototype;
```

Comment:
Like this deprecate was pulled over from `backbone-metal` and it originally used `_.noop` until it was tested against older underscore:  https://github.com/marionettejs/backbone-metal/commit/d16fe23cb2dfe0bd9308a5e2c440c2247a9e7788#diff-2b7c460ee7b16a2db70ff8cc661ee9cdL482